src/main/java/com/pdffiller/client/utils/MultipartHelper.java

```diff
@@ -0,0 +1,88 @@
+package com.pdffiller.client.utils;
+
+import java.io.File;
+import java.io.IOException;
+import java.math.BigInteger;
+import java.nio.file.Files;
+import java.security.SecureRandom;
+import java.util.HashMap;
+
+public class MultipartHelper {
+  private static final String CRLF = "\r\n";
+  private File file;
+  private String boundary;
+  private byte[] message;
+
+  public File getFile() {
+    return file;
+  }
+
+  public void setFile(File file) {
+    this.file = file;
+  }
+
+  public byte[] getMessage() {
+    return message;
+  }
+
+  public MultipartHelper(String file) {
+    this.file = new File(file);
+    boundary = generateBoundary();
```

Comment:
It is not a good style to initialize boundary and message in constructor.
Yes, you can do it, but I would not recommend it because
    - it  makes it harder to create granular unit tests for this class and you don't get immediate feedback 
    - There also some caveats: like you should never call overriden methods in the constructor